meta-mender-tegra/classes/tegra-mender-setup.bbclass

```diff
@@ -19,7 +19,9 @@ ARTIFACTIMG_FSTYPE = "ext4"
 IMAGE_TYPEDEP_tegraflash += " dataimg"
 IMAGE_FSTYPES += "dataimg"
 PREFERRED_PROVIDER_u-boot-fw-utils = "u-boot-fw-utils-tegra"
+PREFERRED_PROVIDER_libubootenv_tegra = "${'libubootenv-fake' if d.getVar('PREFERRED_PROVIDER_virtual/bootloader').startswith('cboot') else 'libubootenv'}"
```

Comment:
We have a bbappend to mender-client that adds an `RDEPENDS` on `libubootenv-fake` for the cboot build. I like what you have here better as it's a more general solution -- can we drop that append with these changes?

Reply:
I was thinking the same thing.   We'd still need the RDEPENDS on `tegra-bup-payload`, anyway, so I left it alone for now.